packages/config/src/context.js

```diff
@@ -19,8 +44,9 @@ const mergeContext = function ({ context: contextProps, ...config }, context, br
     .filter(Boolean)
     .map(normalizeBuildCase)
     .map(addCommandConfigOrigin)
```

Comment:
We should use [`addBuildCommandConfigOrigin`](https://github.com/netlify/build/blob/f7613c320d223833d853c511a7d8ea3de8bdcc83/packages/config/src/origin.js#L24) here, so that `addNamespace()` receives `config` instead of `config.build`. This would allow simplifying `addNamespace()`.